test/support/test_dynamic_load_utils.h

```diff
@@ -17,7 +17,15 @@
 #include <iostream>
 #include "support/test_config.h"
 #if TEST_DYNAMIC_SELECTION_AVAILABLE
-#    include "support/sycl_sanity.h"
+
+namespace TestUtils
+{
+template <typename Op, ::std::size_t CallNumber>
+struct unique_kernel_name;
+
+template <typename Policy, int idx>
+using new_kernel_name = unique_kernel_name<typename ::std::decay_t<Policy>, idx>;
+} // namespace TestUtils
 
```

Comment:
To avoid repetition, can we include test_dynamic_selection_utils.h in test_dynamic_load_utils.h and remove TestUtils?

Reply:
Yes, but there are other functions with similar names which cause a conflict. A clean-up of the tests is eventually to be done.

Reply:
Ok, as a TODO then, we'll reconcile the names and remove the repetition.